tests/Api/Responses/admin/promotion/post_promotion_with_invalid_rules_response.json

```
@@ -4,6 +4,8 @@
     "@type": "@string@.startsWith('ConstraintViolation')",
     "hydra:title": "An error occurred",
     "hydra:description": "@string@",
+    "title": "@string@.contains('An error occurred').optional()",
+    "description": "@string@.contains('endsAt: End date cannot be set prior start date.').optional()",
```

```suggestion
    "description": "@string@.optional()",
```
This message isn’t accurate for the actual test scenario. It looks like it was copied from a different case.